tsdb/chunks/chunks.go

```diff
@@ -311,6 +314,50 @@ func MergeChunks(a, b chunkenc.Chunk) (*chunkenc.XORChunk, error) {
 	return newChunk, nil
 }
 
+// Avoid chunks with more than a certain number of samples
+// Max of 120 samples per chunk results in best compression ratio.
+func BreakLargeChunks(chks []Meta, limit int) ([]Meta, error) {
+	newChunksArray := make([]Meta, 0 , len(chks)*2)
+	for _, c := range chks {
+		currentChunk := c.Chunk
```

Comment:
I wouldn't put this in a var, and instead would just reference `c.Chunk` directly in the two places below this is used. I'm guessing the intent was deduplication, but I don't think it is beneficial in this case because it's just a deref (so not resource intensive) and if the struct member name changes compilation will fail so it's easy to fix, meaning you're just indirecting the name which adds another thing for readers to follow (ie: "what's `currentChunk` again?").

Reply:
Okay will do. Thanks for the explanation.